src/components/AdoptionForm.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react'
+
+const AdoptionForm = props => {
+  const [newApplication, setNewApplication] = useState({
+    name:"", 
+    phoneNumber:"",
+    email:"",
+    homeStatus:""
+  })
+  const handleAppChange = event => {
+    setNewApplication({
+      ...newApplication,
+      [event.currentTarget.name]: event.currentTarget.value
+    })
+  }
+  const handleAppSubmit = event => {

```

Comment:
I would leave a blank line in between each code block. Having them all touch together makes it much harder to read.